### ISSUE TITLE
possible error in the readme

### ISSUE THREAD

**Issue by projectvalis** [quantifind/wisp#32]
shouldn't the dependency read something like:

`com.quantifind % wisp_2.11 % 0.0.4`

?

**Comment by AustinBGibbons** [comment]
Hi! The current line `"com.quantifind" %% "wisp" % "0.0.4` uses `%%` to infer the correct version of scala (we cross compile 2.11 and 2.10) rather than having to specify `"wisp_2.11"`

Do you have issue fetching the dependency?

Check out this SO post: http://stackoverflow.com/questions/17461453/build-scala-and-symbols-meaning

**Comment by projectvalis** [comment]
yeah I did - but I'm using spark-notebook at the moment so that might've had something to do with it. no worries - and good to know about the "%%" !
